addons/mail/static/src/new/messaging_menu/messaging_menu.xml

```diff
@@ -30,8 +30,9 @@
 <t t-name="mail.messaging_menu.preview" owl="1">
     <t t-set="thread" t-value="messaging.threads[preview.id]"/>
     <div class="o-mail-notification-item d-flex cursor-pointer border-bottom align-items-center" t-on-click="() => this.openDiscussion(thread.id)">
-        <div class="m-1" style="width:40px;height:40px;">
+        <div class="position-relative d-flex m-1" style="width:40px;height:40px;">
             <img class="w-100 h-100 rounded-circle" alt="Thread Image" t-att-src="thread.imgUrl" />
+            <PartnerImStatus t-if="thread.type === 'chat'" partner="messaging.partners[thread.chatPartnerId]"/>
```

Comment:
Ok for the moment to do this: manually add img and `PartnerImStatus` (even if partner has no im status, component guards it).

Note that this code doesn't properly link `thread.imgUrl` with `thread.chatPartnerId`, hence why you need some code that we should actively maintain to sync data properly.

It's ok for the moment, but eventually we should do something, because this adds some significant technical debts 😅